packages/web/src/components/notification/NotificationPanel.tsx

```diff
@@ -146,16 +146,14 @@ export const NotificationPanel = ({ anchorRef }: NotificationPanelProps) => {
                 getScrollParent={getScrollParent}
               >
                 <div className={styles.content}>
-                  {notifications
-                    .filter(({ isHidden }: any) => !isHidden)
```

Comment:
that's cool so was this just never used or handled somewhere else?

Reply:
this feature was removed